mne_bids/dig.py

```diff
@@ -150,7 +118,7 @@ def _get_impedances(raw, names):
     return impedances
 
 
-def _electrodes_tsv(raw, fname, datatype, overwrite=False, verbose=True):
+def _write_electrodes_tsv(raw, fname, datatype, overwrite=False, verbose=True):
```

Comment:
why the renaming?

Reply:
Personally I think the new name makes things clearer, but I'd also vote for reverting this change here and make a new PR if this is really important to you

Reply:
I mainly renamed because it was more clear for dev purposes.

Reply:
I don't feel strongly against it, so if it helps you, I'm okay with leaving the new names.
@sappelhoff, your call :)

Reply:
fine with me as well to leave it in the slightly-more-verbose-but-easier-to-understand version 👍